org/mozilla/jss/pkcs11/PK11DSAPrivateKey.java

```diff
@@ -6,7 +6,7 @@
 import java.security.interfaces.DSAParams;
 import java.security.interfaces.DSAPrivateKey;
 
-class PK11DSAPrivateKey
+public class PK11DSAPrivateKey
     extends PK11PrivKey implements DSAPrivateKey
```

Comment:
This is separate from this PR. Can we rename `PK11PrivKey` to `PK11PrivateKey` since it will appear in javadoc? I wouldn't worry about backward compatibility as long as it's done in a new minor version (i.e. 4.x).

Reply:
`PK11PrivKey` [already exists in Javadocs](https://dogtagpki.github.io/jss/master/javadocs/org/mozilla/jss/pkcs11/PK11PrivKey.html), without this change.

While a historical artifact, I think the naming makes some sense: the core classes have `PK11PrivKey` / `PK11PubKey` / `PK11Cert` / `PK11SymKey`, but while they're what gets used internally to JSS most often, as most applications (algorithms which use these keys) are agnostic of actual key type. 

However, occasionally more data is necessary and it is necessary to extract details about the (public) keys. Hence why the end classes are necessary. That happens, e.g., in the JSS Provider SPIs.

Reply:
I'm not sure about symkey or cert, but shortening private and public into priv and pub seems to be uncommon, unnecessary, and inconsistent with the interfaces they're implementing (i.e. `PrivateKey` and `PublicKey`).